src/soc/ibm/power9/homer.c

```diff
@@ -1470,6 +1353,237 @@ static void layout_rings_for_sgpe(struct homer_st *homer,
 	}
 }
 
+static void stop_save_scom(struct homer_st *homer, uint32_t scom_address,
+			   uint64_t scom_data, enum scom_section section,
+			   enum scom_operation operation)
+{
+	enum {
+		ORI_OPCODE = 24,
```

Comment:
Keep it consistent with ATTN_OPCODE 
```suggestion
		ORI_OPCODE = 0x60000000,
```
Also, I already defined ATTN_OP and BLR_OP in lines ~235, maybe move this no-op there and skip the rest.